build/packages.targets

```diff
@@ -19,42 +20,44 @@
+        <PackageDownload Update="Microsoft.VisualStudio.ProjectSystem.Managed" Version="[$(ProjectSystemManagedVersion)]" />
+        <PackageDownload Update="Microsoft.VisualStudio.ProjectSystem.Managed.VS" Version="[$(ProjectSystemManagedVersion)]" />
 
         <PackageDownload Update="NuGet.Core" Version="[$(NuGetCoreV2Version)]" />
         <PackageDownload Update="VSLangProj140" Version="[14.0.25029]" />
         <PackageDownload Update="VSLangProj150" Version="[1.0.0]" />
         <!-- PackageReference -->
         <PackageReference Update="Lucene.Net" Version="3.0.3" />
         <PackageReference Update="MessagePack" Version="2.2.60" />
-        <PackageReference Update="Microsoft.Build" Version="$(MicrosoftBuildPackageVersion)" />
+        <PackageReference Update="Microsoft.Build" Version="$(MicrosoftBuildPackageVersion)" ExcludeAssets="runtime" />
         <PackageReference Update="Microsoft.Build.Framework" Version="$(MicrosoftBuildPackageVersion)" />
         <PackageReference Update="Microsoft.Build.Locator" Version="1.2.6" />
         <PackageReference Update="Microsoft.Build.Tasks.Core" Version="$(MicrosoftBuildPackageVersion)" />
         <PackageReference Update="Microsoft.Build.Utilities.Core" Version="$(MicrosoftBuildPackageVersion)" />
         <PackageReference Update="Microsoft.CodeAnalysis.FxCopAnalyzers" Version="2.9.8" />
         <PackageReference Update="Microsoft.DataAI.NuGetRecommender.Contracts" Version="2.1.0" />
         <PackageReference Update="Microsoft.Extensions.CommandLineUtils" Version="1.0.1" />
-        <PackageReference Update="Microsoft.Internal.VisualStudio.Shell.Embeddable" Version="$(VSFrameworkVersion)" />
+        <PackageReference Update="Microsoft.Internal.VisualStudio.Shell.Embeddable" Version="17.0.0-preview-1-30928-1111" />
         <PackageReference Update="Microsoft.SourceLink.GitHub" Version="1.0.0" />
         <PackageReference Update="Microsoft.ServiceHub.Framework" Version="2.7.327-preview" />
         <PackageReference Update="Microsoft.TeamFoundationServer.ExtendedClient" Version="$(VSServicesVersion)" />
-        <PackageReference Update="Microsoft.VisualStudio.ComponentModelHost" Version="16.10.31" />
+        <PackageReference Update="Microsoft.VisualStudio.ComponentModelHost" Version="17.0.28-g439d20ddd3" />
         <PackageReference Update="Microsoft.VisualStudio.Composition" Version="16.4.11" />
-        <PackageReference Update="Microsoft.VisualStudio.Editor" Version="16.10.31" />
-        <PackageReference Update="Microsoft.VisualStudio.Language.Intellisense" Version="16.10.31" />
-        <PackageReference Update="Microsoft.VisualStudio.Language.StandardClassification" Version="16.10.31" />
+        <PackageReference Update="Microsoft.VisualStudio.Editor" Version="17.0.24-g4265d215ae" />
+        <PackageReference Update="Microsoft.VisualStudio.Imaging.Interop.14.0.DesignTime" Version="14.3.26931" NoWarn="NU1605" />
```

Comment:
NoWarn/ExcludeAssets/IncludeAssets should not be set in the packages.targets file cause it messes with visibility. 

See: https://github.com/NuGet/NuGet.Client/blob/dev/build/packages.targets for example, it contains none of these metadata items.

Reply:
Note that I understand that this is something we're trying to get done sooner than later, so I'm not necessarily asking for  a change, but rather that we should create an issue.

Reply:
@nkolev92 Understood. There are still hacks here. Some of them may not even be needed anymore. To give you some context, I only had my first successful lab build yesterday and I have no way to test these changes in VS. So everything is still rough.

The goal of this PR is just to produce packages that will unblock the interop work of other teams since we're trying to have the interop work all-up completed by EOW. The initial PR in this branch was hacky and this one builds upon that one. There will be more changes coming.

1. I would prefer not to create issues to track the remaining work here, but I will do so if that's what the team wants. In my mind it's all covered by https://github.com/NuGet/Client.Engineering/issues/754 which is not yet complete, even with this PR.
2. I was hoping that once this work was complete I could create a PR to review and merge all of the interop-related changes from this branch into the "real" 6.0.x branch. Perhaps that doesn't make sense given the official-sounding name of this branch though. I wonder if @dtivel and @rrelyea had a long-term branch plan for this work. Perhaps calling the target branch release-6.0.x was a mistake? Or perhaps it was necessary so the right things (release things) would happen when changes were made in this branch.

Reply:
Discussed offline. 
We'll just review these changes in more detail when they are ported into the long term branch.